index.js

```diff
@@ -2,14 +2,10 @@
 
 const ZipArray = xs => ({
   map: f => ZipArray(xs.map(f)),
-  ap: ys => {
-    const zs = ys.runZipArray;
-    let result = [];
```

Comment:
We can use `const` here, since we never reassign to `result`. `const`/`let` are concerned with reassignment but have nothing to say about mutation.